build/azure-pipelines/linux/Dockerfile

```diff
@@ -1,12 +1,17 @@
 #Download base image ubuntu 18.04
 FROM ubuntu:18.04
 
+#Adding apt repos for g++-4.9
```

Comment:
So the process is update Docker file -> push update to the container registry -> update pipeline?

Reply:
Yes, 

1. Update the docker file
1. Build an image 
1. Push to container registry 
1. Update the pipeline. 

I will create a doc for this as well